### ISSUE TITLE
Select multiple date/resources at the same time in fullcalendar scheduler resource timeline

### ISSUE THREAD

**Issue by jvguidio** [fullcalendar/fullcalendar#5015]
Hello,

Is it possible to add a way to select multiple resources (related or broken) at the same time? Maybe using Ctrl + Click or something like this... It would be great if we have a way to do that. 

The image bellow should explain better what I tried to describe.

![image](https://user-images.githubusercontent.com/54725503/64050918-de10ab00-cb4f-11e9-96e5-5dfc8f83f3b7.png)

Thank You!

**Comment by acerix** [comment]
I made a demo showing the selection is limited to one resource.

https://codepen.io/acerix/pen/bGboPwM?&editable=true&editors=001

To confirm, are you looking to allow the selecting to go through multiple adjacent resources?

Your "option 2" could be done by storing a list of the "selected resources" and you could show the previous selections with a background event.

**Comment by jvguidio** [comment]
> To confirm, are you looking to allow the selecting to go through multiple adjacent resources?
Yes, the option 1 is the possibility to go through multiple adjacent resources.

> Your "option 2" could be done by storing a list of the "selected resources" and you could show the previous selections with background events.
I'll try to do that. Do you have any example to provide?

**Comment by jvguidio** [comment]
Hello acerix,

As you suggested, I could do the option 2 using the background events. I could store all the selections in a List, used the background events to fill the 'gaps' in calendar and then used the List of stored event in my back-end to do some other stuff.

Regarding the first option (Option 1), selecting multiple adjacent resources is a great feature that could be implemented, even though I don't know if this feature is a hard to implement.

**Comment by mperktold** [comment]
I would also like to have these features.

Additionally, I would like to be able to toggle between selecting rectangular "blocks" of the same dates in adjacent resources, and selecting adjacent dates in different resources but in a single gesture.

The idea is that it is a single event which needs a single resource, but switches resource in between. An example could be a meeting that needs to move to another room at some point.

![switchResourcesInBetween](https://user-images.githubusercontent.com/7334069/114841510-36a1c400-9dd8-11eb-81fa-f6ecf0c4e000.png)

The controls for toggling between the two modes should be implemented by the application, e.g. holding or not holding `Shift`, but it needs some support from the API.

Probably, this could also be implemented using background events, but maybe not with a single gesture, which makes it harder to ensure that the switch is done without introducing gaps on the time dimension.

**Comment by salmanakber** [comment]
Hello i need to allow only limited selection to user in calander like every one should only be able to select 5 cell not more than 5

**Comment by gordonk** [comment]
+1

**Comment by danceslikesnake** [comment]
+1

**Comment by wootwoot1234** [comment]
+1
